fc/verify.go

```diff
@@ -16,23 +16,26 @@ type VerifySignalsResponse struct {
 }
 
 type VerifyMatchResponse struct {
-	City       bool `json:"city"`
-	Region     bool `json:"region"`
-	Country    bool `json:"country"`
-	Continent  bool `json:"continent"`
-	PostalCode bool `json:"postalCode"`
-	FamilyName bool `json:"familyName"`
-	GivenName  bool `json:"givenName"`
-	Phone      bool `json:"phone"`
-	Email      bool `json:"email"`
-	Maid       bool `json:"maid"`
-	Social     bool `json:"social"`
-	NonId      bool `json:"nonId"`
+	City       bool    `json:"city"`
+	Region     bool    `json:"region"`
+	Country    bool    `json:"country"`
+	Continent  bool    `json:"continent"`
+	PostalCode bool    `json:"postalCode"`
+	FamilyName bool    `json:"familyName"`
+	GivenName  bool    `json:"givenName"`
+	Phone      bool    `json:"phone"`
+	Email      bool    `json:"email"`
+	Maid       bool    `json:"maid"`
+	Social     bool    `json:"social"`
+	NonId      bool    `json:"nonId"`
+	Risk       float64 `json:"risk"`
 }
 
 type VerifyActivityResponse struct {
-	Emails  float64 `json:"emails"`
-	Message string  `json:"message"`
```

Comment:
Similarly, the message field is also missing the doc.
```
{
    "email" : 0.82,
    "online" : 0.91,
    "social" : 0.32,
    "employment" : 0.78
}
```

I've removed the field here. Is the field needed?
https://docs.fullcontact.com/docs/verify-activity